client/container/Guarantee.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState, useEffect } from 'react';
+
+import Guarantee from '../components/Guarantee/Guarantee';
+
+import { fetchFromDB } from '../utils/fetch';
+
+const guaranteeContainer = (props) => {
```

Comment:
@WormJim can you explain the differences between `guaranteeContainer` and `guaranteeComponent` in `client/components/Guarantee/Guarantee.js`, line 4? I am confused.

Reply:
A container is a component except that it contains state.  A component is passed state.